diff -urN gcm-2.0.4.orig/configure.in gcm-2.0.4/configure.in
--- gcm-2.0.4.orig/configure.in	Tue Mar  4 09:19:30 2003
+++ gcm-2.0.4/configure.in	Tue Mar  4 09:34:05 2003
@@ -56,6 +56,10 @@
 localedir=${datadir}/locale
 AC_SUBST(localedir)
 
+GETTEXT_PACKAGE=gcm
+AC_SUBST(GETTEXT_PACKAGE)
+AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE, "$GETTEXT_PACKAGE",[fix])
+
 dnl Set PACKAGE_LOCALE_DIR in config.h.
 if test "x${prefix}" = "xNONE"; then
   AC_DEFINE_UNQUOTED(PACKAGE_LOCALE_DIR, "${ac_default_prefix}/${DATADIRNAME}/locale")
