diff -urN gcm-2.0.1.orig/configure.in gcm-2.0.1/configure.in
--- gcm-2.0.1.orig/configure.in	Fri Sep 27 00:17:59 2002
+++ gcm-2.0.1/configure.in	Fri Sep 27 01:28:02 2002
@@ -90,9 +90,9 @@
 localedir=${datadir}/locale
 AC_SUBST(localedir)
 
-dnl GETTEXT_PACKAGE=gcm
-dnl AC_SUBST(GETTEXT_PACKAGE)
-dnl AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE, "$GETTEXT_PACKAGE")
+GETTEXT_PACKAGE=gcm
+AC_SUBST(GETTEXT_PACKAGE)
+AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE, "$GETTEXT_PACKAGE",[fix])
 
 
 dnl Set PACKAGE_LOCALE_DIR in config.h.
