diff -urN gcm-2.0.4.orig/Makefile.am gcm-2.0.4/Makefile.am
--- gcm-2.0.4.orig/Makefile.am	Tue Mar  4 09:19:30 2003
+++ gcm-2.0.4/Makefile.am	Tue Mar  4 09:22:47 2003
@@ -20,7 +20,7 @@
 
 
 
-desktopdir  = $(datadir)/gnome/apps/Applications/Other/
+desktopdir  = $(datadir)/applications
 desktop_in_files = gcm.desktop.in
 desktop_DATA = $(desktop_in_files:.desktop.in=.desktop)
 
diff -urN gcm-2.0.4.orig/configure.in gcm-2.0.4/configure.in
--- gcm-2.0.4.orig/configure.in	Tue Mar  4 09:19:30 2003
+++ gcm-2.0.4/configure.in	Tue Mar  4 09:24:16 2003
@@ -91,7 +91,7 @@
 dnl Set PACKAGE DIRs in config.h.
 packagepixmapsdir=share/pixmaps/${PACKAGE}
 packagehelpdir=share/gnome/help/${PACKAGE}
-packagemenudir=share/gnome/apps
+packagemenudir=share
 
 dnl Subst PACKAGE_DATA_DIR.
 NO_PREFIX_PACKAGE_DATA_DIR="${packagedatadir}"
